.github/workflows/nightly_detect_failure.yml

```
@@ -49,6 +49,8 @@ jobs:
               printf 'Creating tag %s from the current state of the nightly-testing branch.' "nightly-testing-${version}"
               git tag "nightly-testing-${version}"
               git push origin "nightly-testing-${version}"
+              hash=$(git rev-list -n 1 "nightly-testing-${version}")
```

Mostly irrelevant, but slightly more idiomatic maybe?

```suggestion
              hash=$(git rev-parse "nightly-testing-${version}")
```